File: CS/WinExample.Module/LocalEmployee.cs
using DevExpress.Xpo;

using DevExpress.Persistent.Base;

namespace WinExample.Module {
    [DefaultClassOptions]
    public class LocalEmployee : EmployeeBase {
        public LocalEmployee(Session session) : base(session) { }
        private string insurancePolicyNumber;
        public string InsurancePolicyNumber {
            get { return insurancePolicyNumber; }
            set {
                SetPropertyValue("InsurancePolicyNumber", ref insurancePolicyNumber, value);
            }
        }
    }

}

File: VB/WinExample.Module/LocalEmployee.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.Xpo

Imports DevExpress.Persistent.Base

Namespace WinExample.Module
	<DefaultClassOptions> _
	Public Class LocalEmployee
		Inherits EmployeeBase
		Public Sub New(ByVal session As Session)
			MyBase.New(session)
		End Sub
		Private insurancePolicyNumber_Renamed As String
		Public Property InsurancePolicyNumber() As String
			Get
				Return insurancePolicyNumber_Renamed
			End Get
			Set(ByVal value As String)
				SetPropertyValue("InsurancePolicyNumber", insurancePolicyNumber_Renamed, value)
			End Set
		End Property
	End Class

End Namespace
